### PR TITLE
LPS-110655 change definition for new data engine ddm save model

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**

### CODE REVIEW

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
Just started reviewing :)

File: modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/ddm/edit_ddm_structure.jsp [hunk 1]
```diff
@@ -265,43 +266,26 @@ renderResponse.setTitle(title);
...
-									value: description,
-								},
-								name: {
-									value: name,
-								},
+				Liferay.componentReady('<portlet:namespace />dataLayoutBuilder').then(
+					function(dataLayoutBuilder) {
+						var formData = dataLayoutBuilder.getFormData();
+
+						var dataDefinition = formData.definition;
```
Comment (adolfopa) [inline modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/ddm/edit_ddm_structure.jsp]: Shouldn't this be formData.dataDefinition?

Comment (adolfopa) [reply modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/ddm/edit_ddm_structure.jsp]: Nevermind, I see this is a field of the data engine object, not the form.

**Comment by adolfopa** [comment]
Pull request submitted to https://github.com/adolfopa/liferay-portal/pull/1352.

**Comment by adolfopa** [comment]
Pull request submitted to https://github.com/adolfopa/liferay-portal/pull/1356. [See changes here.](https://github.com/adolfopa/liferay-portal/compare/65425b5b97a1~1...65425b5b97a1)

**Pull request closed without merging**